src/test/ui/wf/wf-in-foreign-fn-decls-issue-80468.rs

```diff
@@ -1,3 +1,4 @@
+// check-pass
```

Comment:
It's surprising that this change is only picked up in *this* test. I would have expected it to be picked up in a parsing or lowering test or something.

Regardless, this change will need a lang (probably) FCP. FWIW, I personally don't think allowing elided lifetimes in `impl` headers is the right move. It's 1) inconsistent with the general trend of not allow elided lifetime parameters and 2) unspecified on if it would be `'static` (as is assumed today under error conditions or some new lifetime (which would be the case if you did `'_`.